### ISSUE TITLE
Signal 4.34.8 - Backups very small, not saving media?

### ISSUE THREAD

**Issue by drolsquito** [signalapp/Signal-Android#8650]
Today I wanted to reset my phone, I did a backup to restore after the reset and the size was less than 10mo, I tried with a friend's phone and same thing... so, maybe I'm wrong but I suppose that Signal backups only messages now? Or is it a bug with the last update? It did full backups 2-3 days ago.

If it saves only messages now, is there a possibility to save media as well?

**Comment by drolsquito** [comment]
Found the problem, it came from the fact my ROM didn't have GAPPS, so what I did was:
- installed a ROM without GAPPS
- installed Signal and restaured a backup
- installed MicroG
- Signal worked fine but couldn't do full backups anymore...
- Uninstalled Signal (+ deleted data and cache)
- Reinstalled Signal and restored previous full backup
- Signal works fine and can do full backups!

So, if you plan to install GAPPS or MicroG on a ROM without, first do a Signal backup, uninstall Signal, install GAPPS or MicroG and install Signal again.

That's it, problem resolved.
